latest-langs

```diff
@@ -63,7 +64,7 @@ for %langs.sort: *.key.fc -> (:key($name), :value(%lang)) {
         my    $res = $ua.get("https://$_")<content>.decode;
 
         when / 'endoflife' / { $res.&from-json[0]<latest> }
-        when / 'github'    / { $res ~~ / [ $name || 'Release' ] ' ' v? <(\d+ <[\d.]>+)> / }
+        when / 'github'    / { $res ~~ / [ $name || 'Release' ] ' ' .+? <(\d+ <[\d.]>+)> / }
```

Comment:
Minor adaptation to the approach for `wikipedia` fixes it, without breakage to others.

Reply:
On closer observation, that ain't entirely true. I will set things straight.